builder/.github/workflows/push-image.yml

```diff
@@ -38,10 +38,9 @@ jobs:
       env:
         PAKETO_BUILDPACKS_DOCKERHUB_USERNAME: ${{ secrets.PAKETO_BUILDPACKS_DOCKERHUB_USERNAME }}
         PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD: ${{ secrets.PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD }}
-        DOCKERHUB_ORG: "paketobuildpacks"
+        GITHUB_REPOSITORY_OWNER: ${{ github.repository_owner }}
       run: |
-        # Strip off the Github org prefix from repo name
-        # paketo-buildpacks/builder-with-some-name --> builder-with-some-name
```

Comment:
nitpick: can we keep the comments for the registry_repo line?

Reply:
yeap, I reverted the comment back :)